src/related-items-and-comparisons/OutfitList.jsx

```diff
@@ -5,27 +5,34 @@ const OutfitList = (props) => {
   // hook for array of outfits, starts with nothing by default
   // items must be unique, so clicking does nothing if product is already in the list
   // should always have an "add to list" card
-  const [outfitIDs, updateOutfitIDs] = useState([]);
-  const [outfit, updateOutfit] = useState([]);
-  const [currList, setCurrList] = useState(outfit);
+  var storedFits = JSON.parse(localStorage.getItem('outfit')) || [];
+  var storedIDs = JSON.parse(localStorage.getItem('outfitIDs')) || [];
+  const [outfitIDs, updateOutfitIDs] = useState(storedIDs);
```

Comment:
Instead of retrieving these from storage and parsing their JSON every time the component renders, you could do something like this:
```javascript
const [outfitIDs, updateOutfitIDs] = useState(() => JSON.parse(localStorage.getItem('outfitIDs')) || []);
```
Same goes for `storedFits`.

Reply:
fixed